readme.md

```diff
@@ -21,8 +21,8 @@ Follow
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_stars/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_stars/report/html_report/)
+- [DEMO LINK](https://Reyti.github.io/layout_stars/)
```

Comment:
please check your deployment since DEMO link leads to readme.md and TEST link leads to 404 page